app/models/participant.rb

```diff
@@ -42,9 +42,11 @@ class Participant < ActiveRecord::Base
   has_many :memberships, dependent: :destroy
   has_many :shift_participants, dependent: :destroy
   has_many :organization_statuses, dependent: :destroy
+  has_many :events
   belongs_to :phone_carrier
   belongs_to :user, dependent: :destroy
 
+
```

Comment:
http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file